docs/index.rst

```diff
@@ -2,42 +2,64 @@
 sunpy core Documentation
 ************************
 
-The sunpy core package is a community-developed, free and open-source solar
+``sunpy`` is a community-developed, free and open-source solar
 data analysis environment for Python.
 
+New Users
+*********
+
+If you are a first time user or new to Python,
+
+..
+        * :Introduction to sunpy core [new page described in #6615]
+
+* :doc:`installation guide <intro/installation>`
+
+Our **tutorial** provides a walkthrough on how to use the key features of sunpy core, we recommend new users start here.
+
+* :doc:`Getting Started <intro/index>`
+
+  #. :doc:`Data Acquisition <intro/acquiring_data/index>`
+  #. :doc:`Data Types <intro/data_types/index>`
+  #. :doc:`Plotting <intro/plotting>`
+  #. :doc:`Units <intro/units>`
+  #. :doc:`Coordinates <intro/coordinates>`
+  #. :doc:`Time <intro/time>`
+
+
+Next Steps
+**********
+
+* :doc:`Example gallery <generated/gallery/index>` contains short-form guides on accomplishing common tasks using sunpy.
+* :doc:`API reference<reference/index>` contains a technical description of the functionality of sunpy core's modules. It describes how they work and how to use them but assume you have a good understanding of sunpy and the scientific python ecosystem in general.
```

Comment:
```suggestion
* :doc:`API reference<reference/index>` contains a technical description of the inputs, outputs, and behaviour of each component of sunpy core.
```
I don't think API ref requires a good understanding of sunpy? With a bit of knoweldge (but not good understanding) I'd expect to be able to understand API reference entries in isolation.